src/test/java/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,40 @@
+package hw3.pages;
+
+import hw3.pages.components.DifferentElementsInputFieldsComponent;
+import hw3.pages.components.DifferentElementsLogsComponent;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+public class DifferentElementsPage extends AbstractPage {
+    DifferentElementsLogsComponent differentElementsLogsComponent;
+    DifferentElementsInputFieldsComponent differentElementsInputFieldsComponent;
+
+    public DifferentElementsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    void initElementsOnPage() {
+        differentElementsLogsComponent = new DifferentElementsLogsComponent(driver);
+        differentElementsInputFieldsComponent = new DifferentElementsInputFieldsComponent(driver);
+    }
+
+    public boolean logSectionContains(List<String> expectedlogs) {
+        return differentElementsLogsComponent.logSectionContains(expectedlogs);
+    }
+    public void clickWater(){
+        differentElementsInputFieldsComponent.clickWater();
+    }
+    public void clickWind(){
+        differentElementsInputFieldsComponent.clickWind();
+    }
+    public void selectSelen(){
+        differentElementsInputFieldsComponent.selectSelen();
+    }
+    public void selectYellow(){
+        differentElementsInputFieldsComponent.selectYellow();
+    }
```

Comment:
Why do you create specific methods for 
* checkbox
* radio button
* select?

I think you could have one method for each element and set name through the parameter